src/main/resources/Dockerfile

```diff
@@ -0,0 +1,173 @@
+##
```

Comment:
This file is a template which goes through the SimpleTemplating engine. I suggest we figure out what the standard suffix is for that type of template. At the very least `Dockerfile.template`